### PR TITLE
copied slot text

### LINKED ISSUE #113: DRAFT: simplify copying a slot to tactical google docs script
a button near the slot title that copies all the items in the slot to the clipboard as plain text

### PR DESCRIPTION
**Pull request by bhavinsen**
Fixed #113

### CODE REVIEW

**Review by chrisns — APPROVED** [review]

**Comment by chrisns** [comment]
@mrjamieeast have you tried this?

**Review by chrisns — CHANGES_REQUESTED** [review]
please write a test for this

**Comment by mrjamieeast** [comment]
This pastes the results upside down (bottom to top) rather than the way they appear on the page

**Comment by mrjamieeast** [comment]
This works for me now. thanks

File: cypress/integration/podcasts.spec.js [hunk 1]
```diff
@@ -105,6 +105,18 @@ describe("newsdesk logged in", () => {
...
+    cy.contains("dev sandbox").click()
+
+    cy.get("section[slotno=7] textarea").type("new items{enter}", {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should("contain", "new items")
+    cy.get("section[slotno=7] button[title='Copy Slot item']").click()
+    cy.get("section[slotno=7]").should("contain", "new items")
+    cy.get("section[slotno=7] button[title='Delete']").click()
+  })
```
Comment (chrisns) [inline cypress/integration/podcasts.spec.js:119]: you should be able to check the clipboard with something like https://egghead.io/blog/handling-copy-and-paste-in-cypress

**Review by chrisns — CHANGES_REQUESTED** [review]

**Review by chrisns — APPROVED** [review]

**Pull request merged**